client/app/components/packages/pas-form/edit.hbs

```diff
@@ -693,35 +693,21 @@
     <div class="cell large-4">
 
       <div class="sticky-sidebar">
-        <p>
-          <button
-            type="button"
-            class="button expanded large"
-            disabled={{not this.isSaveable}}
-            {{on "click" (perform this.save)}}
-            data-test-save-button
-          >
-            {{!-- TODO: Make this button work with the action passed in @save--}}
-            {{#if this.save.isRunning}}
-              <FaIcon @icon="spinner" @spin="true" size="5x" class="light-gray" />
-            {{else}}
-              Save
-            {{/if}}
-          </button>
-          <button
-            type="button"
-            class="button expanded large secondary"
-            disabled={{not this.isSubmittable}}
-            {{on "click" this.toggleModal}}
-            data-test-submit-button
-          >
-            {{#if this.submit.isRunning}}
-              <FaIcon @icon="spinner" @spin="true" size="5x" class="light-gray" />
-            {{else}}
-              Submit
-            {{/if}}
-          </button>
-        </p>
+        <Packages::save-button
+          @clickEvent={{perform this.save}}
```

Comment:
can we name this the same convention we have? `onClick`?

Reply:
👍 Addressed in 5042596.